portfolio/src/main/webapp/script.js

```diff
@@ -54,6 +54,6 @@ async function getComments() {
 
 function createListElement(text) {
     const liElement = document.createElement('li');
-    liElement.innerText = text;
+    liElement.innerText = text.comment + " (The comment's sentiment score is " + (String)(text.sentiment_score) + ")";
```

Comment:
As the score is just to the end of the comment, how about make it more concise?

liElement.innerText = text.comment + " (Sentiment score: " + (String)(text.sentiment_score) + ")";